### PR TITLE
Avoiding use "".format()

### PR DESCRIPTION
**Pull request by KarenKawaii**
``` bash
************* Module addons-vauxoo.account_anglo_saxon_stock_move.model.invoice
account_anglo_saxon_stock_move/model/invoice.py:108: [R8111(prefer-other-formatting), AccountInvoice.reconcile_stock_accrual]
************* Module addons-vauxoo.mail_add_followers_multirecord.wizard.add_followers
mail_add_followers_multirecord/wizard/add_followers.py:51: [R8111(prefer-other-formatting), InviteWizard.default_get]
************* Module addons-vauxoo.mrp_partial_production.models.mrp
mrp_partial_production/models/mrp.py:78: [R8111(prefer-other-formatting), MrpProduction.get_qty_available_to_produce]
************* Module addons-vauxoo.mrp_workcenter_account_move.model.mrp
mrp_workcenter_account_move/model/mrp.py:45: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
mrp_workcenter_account_move/model/mrp.py:63: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
mrp_workcenter_account_move/model/mrp.py:68: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
mrp_workcenter_account_move/model/mrp.py:77: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
mrp_workcenter_account_move/model/mrp.py:84: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
mrp_workcenter_account_move/model/mrp.py:88: [R8111(prefer-other-formatting), MrpProduction.test_accounting_setting]
************* Module addons-vauxoo.product_extended_segmentation.model.template
product_extended_segmentation/model/template.py:63: [R8111(prefer-other-formatting), ProductProduct.update_material_cost_on_zero_segmentation]
product_extended_segmentation/model/template.py:291: [R8111(prefer-other-formatting), ProductTemplate.ensure_change_price_log_messages]
product_extended_segmentation/model/template.py:301: [R8111(prefer-other-formatting), ProductTemplate.ensure_change_price_log_messages]
************* Module addons-vauxoo.product_extended_segmentation.tests.test_avg_costs
product_extended_segmentation/tests/test_avg_costs.py:114: [R8111(prefer-other-formatting), TestAvgCosts.test_02_compute_price_with_real_bom]
************* Module addons-vauxoo.product_extended_segmentation.tests.test_wizard
product_extended_segmentation/tests/test_wizard.py:60: [R8111(prefer-other-formatting), TestWizard.check_default_cost]
product_extended_segmentation/tests/test_wizard.py:73: [R8111(prefer-other-formatting), TestWizard.test_00_test_wizard_defaults]
product_extended_segmentation/tests/test_wizard.py:80: [R8111(prefer-other-formatting), TestWizard.test_00_test_wizard_defaults]
product_extended_segmentation/tests/test_wizard.py:108: [R8111(prefer-other-formatting), TestWizard.test_01_test_threshold_no_update]
product_extended_segmentation/tests/test_wizard.py:172: [R8111(prefer-other-formatting), TestWizard.test_02_use_product_variant_wizard_with_greater_new_price]
product_extended_segmentation/tests/test_wizard.py:191: [R8111(prefer-other-formatting), TestWizard.test_03_use_product_variant_wizard_with_lesser_new_price]
************* Module addons-vauxoo.product_extended_variants.model.template
product_extended_variants/model/template.py:143: [R8111(prefer-other-formatting), ProductTemplate.do_change_standard_price]
************* Module addons-vauxoo.product_extended_variants.wizard.wizard_price
product_extended_variants/wizard/wizard_price.py:76: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:81: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:82: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:89: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:107: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:116: [R8111(prefer-other-formatting), WizardPrice.execute_cron]
product_extended_variants/wizard/wizard_price.py:166: [R8111(prefer-other-formatting), WizardPrice.__execute_cron]
product_extended_variants/wizard/wizard_price.py:172: [R8111(prefer-other-formatting), WizardPrice.__execute_cron]
product_extended_variants/wizard/wizard_price.py:217: [R8111(prefer-other-formatting), WizardPrice.__execute_cron]
************* Module addons-vauxoo.purchase_changeless_move_lines.model.stock_picking
purchase_changeless_move_lines/model/stock_picking.py:169: [R8111(prefer-other-formatting), StockPicking.line2str]
purchase_changeless_move_lines/model/stock_picking.py:178: [R8111(prefer-other-formatting), StockPicking.line2str]
purchase_changeless_move_lines/model/stock_picking.py:185: [R8111(prefer-other-formatting), StockPicking.line2str]
************* Module addons-vauxoo.purchase_requisition_currency.model.purchase_requisition
purchase_requisition_currency/model/purchase_requisition.py:83: [R8111(prefer-other-formatting), PurchaseRequisition.make_purchase_order]
************* Module addons-vauxoo.stock_accrual_report.wizard.wizard
stock_accrual_report/wizard/wizard.py:380: [R8111(prefer-other-formatting), StockAccrualWizard._get_lines_from_stock]
stock_accrual_report/wizard/wizard.py:397: [R8111(prefer-other-formatting), StockAccrualWizard._get_lines_from_invoice]
stock_accrual_report/wizard/wizard.py:414: [R8111(prefer-other-formatting), StockAccrualWizard._get_orders_from_stock]
stock_accrual_report/wizard/wizard.py:431: [R8111(prefer-other-formatting), StockAccrualWizard._get_orders_from_invoice]
************* Module addons-vauxoo.stock_card.model.stock_card
stock_card/model/stock_card.py:64: [R8111(prefer-other-formatting), StockCardProduct._get_quant_values]
************* Module addons-vauxoo.stock_card.tests.test_stock_card
stock_card/tests/test_stock_card.py:101: [R8111(prefer-other-formatting), TestStockCard.test_01_stock_card]
************* Module addons-vauxoo.stock_card.tests.test_stock_card_negative_stock
stock_card/tests/test_stock_card_negative_stock.py:131: [R8111(prefer-other-formatting), TestStockCardNegativeStock.create_purchase_order]
stock_card/tests/test_stock_card_negative_stock.py:148: [R8111(prefer-other-formatting), TestStockCardNegativeStock.create_sale_order]
stock_card/tests/test_stock_card_negative_stock.py:185: [R8111(prefer-other-formatting), TestStockCardNegativeStock.test_01_do_inouts]
stock_card/tests/test_stock_card_negative_stock.py:190: [R8111(prefer-other-formatting), TestStockCardNegativeStock.test_01_do_inouts]
stock_card/tests/test_stock_card_negative_stock.py:195: [R8111(prefer-other-formatting), TestStockCardNegativeStock.test_01_do_inouts]
stock_card/tests/test_stock_card_negative_stock.py:200: [R8111(prefer-other-formatting), TestStockCardNegativeStock.test_01_do_inouts]
************* Module addons-vauxoo.stock_card.tests.test_stock_card_product_returns
stock_card/tests/test_stock_card_product_returns.py:90: [R8111(prefer-other-formatting), TestStockCardProductReturns.test_01_do_inouts]
stock_card/tests/test_stock_card_product_returns.py:97: [R8111(prefer-other-formatting), TestStockCardProductReturns.test_01_do_inouts]
stock_card/tests/test_stock_card_product_returns.py:105: [R8111(prefer-other-formatting), TestStockCardProductReturns.test_01_do_inouts]
stock_card/tests/test_stock_card_product_returns.py:113: [R8111(prefer-other-formatting), TestStockCardProductReturns.test_01_do_inouts]
************* Module addons-vauxoo.stock_card_segmentation.tests.test_stock_card_negative_stock
stock_card_segmentation/tests/test_stock_card_negative_stock.py:92: [R8111(prefer-other-formatting), TestStockCardNegativeStock.create_sale_order]
************* Module addons-vauxoo.stock_landed_costs_average.model.stock_landed_costs
stock_landed_costs_average/model/stock_landed_costs.py:170: [R8111(prefer-other-formatting), StockLandedCost._create_deviation_account_move_line]
stock_landed_costs_average/model/stock_landed_costs.py:183: [R8111(prefer-other-formatting), StockLandedCost._create_deviation_account_move_line]
stock_landed_costs_average/model/stock_landed_costs.py:333: [R8111(prefer-other-formatting), StockLandedCost._create_cogs_account_move_line]
stock_landed_costs_average/model/stock_landed_costs.py:348: [R8111(prefer-other-formatting), StockLandedCost._create_cogs_account_move_line]
************* Module addons-vauxoo.stock_landed_costs_average.tests.test_landed_cost_average
stock_landed_costs_average/tests/test_landed_cost_average.py:80: [R8111(prefer-other-formatting), TestLandedCostAverage.test_02_onchange_invoice_ids]
************* Module addons-vauxoo.stock_landed_costs_average.tests.test_landed_cost_revert
stock_landed_costs_average/tests/test_landed_cost_revert.py:106: [R8111(prefer-other-formatting), TestLandedCostRevert.test_01_landed_cost_revert]
stock_landed_costs_average/tests/test_landed_cost_revert.py:130: [R8111(prefer-other-formatting), TestLandedCostRevert.validate_acct_entries_values]
************* Module addons-vauxoo.stock_landed_costs_average.tests.test_stock_landed_common
stock_landed_costs_average/tests/test_stock_landed_common.py:76: [R8111(prefer-other-formatting), TestStockLandedCommon.create_purchase_order]
stock_landed_costs_average/tests/test_stock_landed_common.py:96: [R8111(prefer-other-formatting), TestStockLandedCommon.create_sale_order]
************* Module addons-vauxoo.stock_purchase_check_fulfillment.wizard.stock_partial_picking
stock_purchase_check_fulfillment/wizard/stock_partial_picking.py:64: [R8111(prefer-other-formatting), StockPartialPicking.do_partial]
stock_purchase_check_fulfillment/wizard/stock_partial_picking.py:97: [R8111(prefer-other-formatting), StockPartialPicking.do_partial]
stock_purchase_check_fulfillment/wizard/stock_partial_picking.py:110: [R8111(prefer-other-formatting), StockPartialPicking.do_partial]
************* Module addons-vauxoo.stock_quant_cost_segmentation.model.stock
stock_quant_cost_segmentation/model/stock.py:359: [R8111(prefer-other-formatting), StockQuant._quant_create]
stock_quant_cost_segmentation/model/stock.py:366: [R8111(prefer-other-formatting), StockQuant._quant_create]
stock_quant_cost_segmentation/model/stock.py:385: [R8111(prefer-other-formatting), StockQuant._quant_create]
************* Module addons-vauxoo.user_story.model.timesheet
user_story/model/timesheet.py:70: [R8111(prefer-other-formatting), AnalyticLine._get_analytic_from_task]
************* Module addons-vauxoo.user_story.model.user_story
user_story/model/user_story.py:83: [R8111(prefer-other-formatting), UserStory.write]
user_story/model/user_story.py:359: [R8111(prefer-other-formatting), UserStory.get_body_disapproval]
user_story/model/user_story.py:364: [R8111(prefer-other-formatting), UserStory.get_body_disapproval]
user_story/model/user_story.py:381: [R8111(prefer-other-formatting), UserStory.get_body_approval]
user_story/model/user_story.py:386: [R8111(prefer-other-formatting), UserStory.get_body_approval]
user_story/model/user_story.py:406: [R8111(prefer-other-formatting), UserStory.do_approval]
user_story/model/user_story.py:495: [R8111(prefer-other-formatting), AcceptabilityCriteria.get_body_disapproval]
************* Module addons-vauxoo.user_story.tests.test_user_story
user_story/tests/test_user_story.py:280: [R8111(prefer-other-formatting), TestUserStory.test_acceptability_criterial_buttons]
user_story/tests/test_user_story.py:304: [R8111(prefer-other-formatting), TestUserStory.test_acceptability_criterial_buttons]
```

Please, just fix the modules that use our customer currently

### CODE REVIEW

**Comment by KarenKawaii** [comment]
## [VX#5832](https://www.vauxoo.com/web#id=5832&view_type=form&model=project.task&menu_id=136&action=138)

PRs dummy:
- lodigroup: https://github.com/Vauxoo/lodigroup/pull/603
- instance: https://github.com/Vauxoo/instance/pull/303
- yoytec (just to not left behind): https://github.com/Vauxoo/yoytec/pull/1914

@moylop260 could you check it please?

Summary:
For lodigroup the tests related to the changes in `addons-vauxoo` were ok, just one failing test in `l10n_mx_cfdi_test` but is not related to `addons-vauxoo` Runbot is running but with warnings.

Instance has the the runbot running with warnings. Travis is green.

Yoytec (It doesn't have any module changed in this PR) But the last runbot was green and in Travis only failed in phantomjs tests.

Regards.

File: mrp_workcenter_account_move/model/mrp.py [hunk 1]
```diff
@@ -39,33 +39,33 @@ def test_accounting_setting(self):
...
                         'Production Cost Journal Entries\n')
-        msg_location = _('Add Financial Account on Location: {location} '
-                         'For product: {product}\n')
+        msg_location = _('Add Financial Account on Location: %(location)s '
+                         'For product: %(product)s\n')
 
         if not self.product_id.property_stock_production.\
                 valuation_in_account_id:
-            msg += msg_location.format(
+            msg += msg_location % (dict(
```
Comment (moylop260) [inline mrp_workcenter_account_move/model/mrp.py]: If you use a parameter you can avoid use `... % (dict(` use just `... % dict(`

File: product_extended_segmentation/model/template.py [hunk 2]
```diff
@@ -298,13 +295,13 @@ def ensure_change_price_log_messages(self, vals):
...
         else:
             subject += 'I cowardly did not update cost, Standard new\n'
-            body += "Price is {comp_th:.2f}% less than old price\n"\
-                "new {new} old {old} \n"\
-                "(current max allowed is {max_th})\n"\
-                .format(old=vals['current_price'],
-                        new=new_price['standard_price'],
-                        comp_th=vals['computed_th'],
-                        max_th=vals['current_bottom_th'])
+            body += "Price is %(comp_th)s.2f%% less than old price\n"\
```
Comment (moylop260) [inline product_extended_segmentation/model/template.py]: The result of `%(comp_th)s.2f%%` vs `{comp_th:.2f}%` using a floating number is very different.
Could you check it?

Comment (moylop260) [inline product_extended_segmentation/model/template.py]: I mean

``` python
print '{comp_th:.2f}%'.format(comp_th=90.123456)
# '90.12%'
print "%(comp_th)s.2f%%" % dict(comp_th=90.123456)
# '90.123456.2f%'
```

Comment (moylop260) [inline product_extended_segmentation/model/template.py]: The right sentence is: `"%(comp_th).2f%%" % dict(comp_th=90.123456)`
to return `90.12%`

File: product_extended_variants/model/template.py [hunk 3]
```diff
@@ -100,10 +100,13 @@ def get_product_accounts(self, cr, uid, product_id, context=None):
                                   context=context)
         product_brw = self.browse(cr, uid, product_id)
         # noqa
-        diff_acc_id = product_brw.property_account_creditor_price_difference and \
+        diff_acc_id = product_brw.\
```
Comment (moylop260) [inline product_extended_variants/model/template.py:103]: Why we need this refactory in this PR?

Comment (KarenKawaii) [reply product_extended_variants/model/template.py:103]: Is only to repair flake8, nothing else.

File: product_extended_segmentation/model/template.py [hunk 4]
```diff
@@ -298,13 +295,13 @@ def ensure_change_price_log_messages(self, vals):
...
-            body += "Price is {comp_th:.2f}% less than old price\n"\
-                "new {new} old {old} \n"\
-                "(current max allowed is {max_th})\n"\
-                .format(old=vals['current_price'],
-                        new=new_price['standard_price'],
-                        comp_th=vals['computed_th'],
-                        max_th=vals['current_bottom_th'])
+            body += "Price is %(comp_th).2f%% less than old price\n"\
+                "new %(new)s old %(old)s \n"\
+                "(current max allowed is %(max_th)s)\n" % (dict(
```
Comment (moylop260) [inline product_extended_segmentation/model/template.py]: Again, remove the first `(` of `(dict(` (in all cases)

File: user_story/model/user_story.py [hunk 5]
```diff
@@ -378,13 +378,13 @@ def do_disapproval(self, cr, uid, ids, context=None):
...
         return _(u'''<html><div>
-                 <h2>{usname}</h2>
-                 <p>The user {user} has approved the user Story
-                 <a href="{link}">See what we are talking about here</a>
-                 </div></html>'''.format(usname=usname, user=username,
-                                         link=link))
+                 <h2>%(usname)s</h2>
+                 <p>The user %(user)s has approved the user Story
+                 <a href="%(link)s">See what we are talking about here</a>
+                 </div></html>''' % (dict(usname=usname,
```
Comment (moylop260) [inline user_story/model/user_story.py]: same case here.
You should search them with `% (dict(`

Comment (KarenKawaii) [reply user_story/model/user_story.py]: @moylop260 Did you mean?:
`% dict(`
or not?

Comment (moylop260) [reply user_story/model/user_story.py]: The focus is search the bad cases
How to search bad cases? `% (dict(`
How to fix them? `% dict(`

File: stock_accrual_report/wizard/wizard.py [hunk 6]
```diff
@@ -59,54 +59,56 @@
...
     sm.state IN ('done')
-    AND (sm.date BETWEEN '{date_start}' AND '{date_stop}')
-    AND sm.company_id = {company_id}
+    AND (sm.date BETWEEN %(date_start)s AND %(date_stop)s)
+    AND sm.company_id = %(company_id)s
 '''
 
 QUERY_INVOICE = '''
 SELECT DISTINCT order_id
-FROM {ttype}_order_line AS ol
```
Comment (moylop260) [inline stock_accrual_report/wizard/wizard.py:99]: The section of `FROM %s` and `INNER JOIN %s` is not supported by `cr.execute(args, values)`

Comment (moylop260) [inline stock_accrual_report/wizard/wizard.py:99]: Check the last comment from http://initd.org/psycopg/docs/usage.html#passing-parameters-to-sql-queries

`...Only variable values should be bound via this method: it shouldn’t be used to set table or field names. For these elements, ordinary string formatting should be used before running execute()`

**Comment by moylop260** [comment]
@dsabrinarg 
Could you review if we don't add a new issue here in lodi?

**Pull request merged**